### PR TITLE
Add optimism delegated manager system deploy script [SIM-64]

### PR DESCRIPTION
**Pull request by pblivin0x**
Notes
- Add deploy script for `ManagerCore`, `DelegatedManagerFactory`, `IssuanceExtension`, `StreamingFeeSplitExtension`, and `TradeExtension`
- Add set-protocol-v2 dependency addresses for `IssuanceModule` and `TradeModule`
- Add new `IssuanceModule` [testnet deployment address](https://github.com/SetProtocol/set-v2-deployments/pull/177)

### CODE REVIEW

File: optimism/deployments/utils/dependencies.ts [hunk 1]
```diff
@@ -171,6 +171,20 @@ export default {
...
+    },
+    69: "0x7853dD7De469638CA26bC9b6f343820be97E5faE",
+  },
+  TRADE_MODULE: {
+    10: {
+      staging: "0x4F70287526ea9Ba7e799D616ea86635CdAf0de4F",
+      production: "0x7215f38011C3e4058Ca3cF7d2b99033016EeFBD8"
+    },
+    69: "0x0a18Ad43a9dc991dB9401ee7aC5AAD0A3b219159",
+  },
```
Comment (cgewecke) [inline optimism/deployments/utils/dependencies.ts:187]: LGTM 💯

File: optimism/deploy/003_delegated_manager_system.ts [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+
+  const streamingFeeModuleAddress = await findDependency(STREAMING_FEE_MODULE);
+  await deployStreamingFeeSplitExtension();
+  const streamingFeeSplitExtensionAddress = await getContractAddress(CONTRACT_NAMES.STREAMING_FEE_SPLIT_EXTENSION);
+
+  const tradeModuleAddress = await findDependency(TRADE_MODULE);
+  await deployTradeExtension();
+  const tradeExtensionAddress = await getContractAddress(CONTRACT_NAMES.TRADE_EXTENSION);
+
+  await initializeManagerCore();
```
Comment (cgewecke) [inline optimism/deploy/003_delegated_manager_system.ts:69]: Same open question here about transferring ownership of ManagerCore to the governance multisig for production...I guess Polygon script will need the same update if this is necessary.

**Review by cgewecke — COMMENTED** [review]
Looks good! Just need to check on governance multisig question...

File: optimism/deploy/003_delegated_manager_system.ts (hunk 2 above)
Comment (pblivin0x) [reply optimism/deploy/003_delegated_manager_system.ts:69]: Added `ManagerCore` ownership transfer to multisig. Also removed unnecessary deferred transaction in `ManagerCore.initialize`.

**Review by cgewecke — APPROVED** [review]
Great! 🏅

**Pull request merged**